tests/testthat/_snaps/bump_version.md

```diff
@@ -26,31 +26,12 @@
     Output
       NULL
 
-# bump_version() works (CRAN rlang)
```

Comment:
rlang was released

Reply:
Do we have a versioned dependency on rlang >= 0.4.12 in `DESCRIPTION`?

Reply:
Now we do, good catch :grin: